srcpkgs/python3-msgpack/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'python3-msgpack'
 pkgname=python3-msgpack
-version=1.0.4
-revision=2
+version=1.0.5
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
 depends="python3"
 checkdepends="python3-pytest"
-short_desc="Python module for Msgpack serializer (Python3)"
+short_desc="Python module for Msgpack serializer"
 maintainer="skmpz <dem.procopiou@gmail.com>"
 license="Apache-2.0"
-homepage="https://msgpack.org/"
```

Comment:
Why change the homepage?

Reply:
The original one is a general homepage for msgpack, not for python3-msgpack specifically. If that's not a good enough justification, I'm fine with changing it back.

Reply:
That's the homepage listed on the Github page

Reply:
Still, when I'm looking for information about python3-msgpack specifically, https://msgpack.org is less than ideal. I don't see a reason to blindly use the homepage listed on the GitHub page.